relay-server/src/actors/global_config.rs

```diff
@@ -256,8 +265,13 @@ impl GlobalConfigService {
         match result {
             Ok(Ok(config)) => {
                 let mut success = false;
+                // No global status implies an outdated upstream, in that case, we fall back on
```

Comment:
Could we make this comment a bit clearer? "outdated" here refers to any relay that doesn't have a ready global config and not an "old" relay, and this can happen in new relays too (e.g. spin up a new relay that can't connect to upstream because the network is down).

Reply:
A missing `global_status` field in the response does indicate the relay (or sentry) we are talking to is on an older version (pre `global_status` field). An updated version would send a default config but also the status as `pending`. But I agree maybe we can phrase this better.

Reply:
Oh right, I confused the status in the global config with the state of relay with a global config. Thanks for pointing it out!